src/java/org/sakaiproject/feedback/tool/entityproviders/FeedbackEntityProvider.java

```diff
@@ -136,6 +139,8 @@ private String handleReport(final EntityView view, final Map<String, Object> par
         final String screenSize = screenWidth + " x " + screenHeight + " pixels";
         final String plugins = (String) params.get("plugins");
         final String ip = requestGetter.getRequest().getRemoteAddr();
+        final String workerNode = ServerConfigurationService.getServerId();
+        final String currentTime = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss").format(new Date());
```

Comment:
Should probably use a locale aware formatter based on the default locale of the server.